nautobot/docs/additional-features/caching.md

```diff
@@ -138,7 +139,8 @@ CACHEOPS_SENTINEL = {
     "db": 0,
     "locations": [
         ("mysentinel.redis.example.com", 26379),
```

Comment:
it is possible to use a DNS entry here that resolves to the 3 sentinel servers as well